ocrd/constants.py

```diff
@@ -0,0 +1,98 @@
+
+NAMESPACES = {
+    'mets': "http://www.loc.gov/METS/",
+    'mods': "http://www.loc.gov/mods/v3",
+    'xlink': "http://www.w3.org/1999/xlink",
+    'page': "http://schema.primaresearch.org/PAGE/gts/pagecontent/2017-07-15",
+    'xsl': 'http://www.w3.org/1999/XSL/Transform#',
```

Comment:
Support for ALTO was seen as highly useful given current practices in libraries and possible uses in OCR-D post-processing of existing OCR.  Current version is [3.1](https://www.loc.gov/standards/alto/v3/alto-3-1.xsd) with 4.0 due for release in April 2018.

Reply:
Agreed, ALTO should be supported. From a programmer's POV it doesn't seem that hard since most features should be easy to translate between the two, i.e. you could use the same API and change the XML handling in the implementation. The same probably goes for hOCR BTW. But we need to discuss the specifics of how we support this and the repercussions on performance and long-term archival aspects etc.